tests/informatics/workflows/test_predictor_evaluation_workflow.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from citrine.informatics.predictor_evaluator import CrossValidationEvaluator
+from citrine.informatics.workflows import PredictorEvaluationWorkflow
+
+
+@pytest.fixture()
+def pew():
+    evaluator = CrossValidationEvaluator(name="test", responses={"foo"})
+    pew = PredictorEvaluationWorkflow(
+        name="Test",
+        description="TestWorkflow",
+        evaluators=[evaluator]
+    )
+    return pew
+
+
+def test_dump(pew):
+    assert pew.dump()["name"] == "Test"
```

Comment:
Wouldn't it be more thorough (and just as easy) to test round-trip de/ser?

Reply:
No, since there are fields that are required when deserializing (like status) that aren't serializable.  Deserialization is covered in `resources/test_predictor_evaulation_workflow.py`.

This testing pattern definitely needs some work, but I think this PR sticks to the style that is in place.